crates/proto/src/error.rs

```diff
@@ -351,7 +351,7 @@ pub struct ForwardNSData {
     /// The referant NS record
     pub ns: Record,
     /// Any glue records associated with the referant NS record.
-    pub glue: Vec<Record>,
+    pub glue: Box<[Record]>,
```

Comment:
Why not make this `Arc`, too?

Reply:
I just changed this to Arc. I don't know that it matters much either way, but my original theory was a) that the glue records are unlikely to ever be cloned separately from the ForwardNSData struct, and probably wouldn't benefit from Arc (vis-à-vis Box) and b) it wouldn't necessarily be more consistent to use Arc since a bunch of other stuff in proto error is boxed.